Header header

uint32 device_id
bool auton_state
uint32 battery_level
uint32 error
